frontend/src/fixtures/personalScheduleFixtures.js

```diff
@@ -0,0 +1,64 @@
+const personalScheduleFixtures = {
+    threePersonalSchedules:
+    [
+        {
+          "id": 1,
+          "user": {
+            "id": 1,
+            "email": "phtcon@ucsb.edu",
+            "googleSub": "115856948234298493496",
+            "pictureUrl": "https://lh3.googleusercontent.com/-bQynVrzVIrU/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucmkGuVsELD1ZeV5iDUAUfe6_K-p8w/s96-c/photo.jpg",
+            "fullName": "Phill Conrad",
+            "givenName": "Phill",
+            "familyName": "Conrad",
+            "emailVerified": true,
+            "locale": "en",
+            "hostedDomain": "ucsb.edu",
+            "admin": true
+          },
+          "description": "Advanced Applications Programming",
+          "quarter": "Winter",
```

Comment:
Quarter should be stored in the format YYYYQ, always.

Also, quarter should contain the year.

Please consider replacing "Winter" with "20221" for example.